pkg/ucauth/kratos/kratos.yml

```diff
@@ -1,6 +1,4 @@
-version: v0.7.1-alpha.1
-
-dsn: memory
+version: v0.8.3-alpha.1.pre.0
```

Comment:
shall we update docker-compose.yml?

Reply:
yes, I will test quick-start and update in a new pr.